### PR TITLE
Add support for k8s patch versions: 1.19.12.1 and 1.20.8.1

### PR DESCRIPTION
**Pull request by ganeshkumar5699**
This change adds support for kubernetes patch versions: 1.19.12.1 and 1.20.8.1.

### CODE REVIEW

**Review by adilliadil — APPROVED** [review]
lgtm

**Review by anujmaheshwari1 — APPROVED** [review]

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -370,11 +370,11 @@ KUBE_BINARY_VERSIONS="
 1.19.7-hotfix.20210310
 1.19.9-hotfix.20210505
 1.19.11
-1.19.12
+1.19.12.1
```
Comment (AbelHu) [inline vhdbuilder/packer/install-dependencies.sh:373]: Is this the k8s official patch release, or a AKS private patch release? I am curious why the version pattern is different, neither 1.19.12 nor 1.19.12-hotfix.2021mmdd?
Should we also apply it to Windows nodes?

**Review by AbelHu — APPROVED** [review]

File: vhdbuilder/packer/install-dependencies.sh (hunk 1 above)
Comment (ganeshkumar5699) [reply vhdbuilder/packer/install-dependencies.sh]: Asked @sozercan and learned that it is a patch release and not a hotfix and that's why it's in this different format. I'm not sure if it's applicable for Windows nodes. @andyliuliming or @sozercan, could you answer this? Thanks!

**Comment by andyliuliming** [comment]
where's the test side change?

**Pull request merged**